.github/workflows/Godot CI.yml

```diff
@@ -0,0 +1,30 @@
+name: Build Island.zip Artifact
+
+on: [push, pull_request, workflow_dispatch] 
+       
+env: 
+  GODOT_VERSION: 3.4.4 
+  DIR: Mod # directory where the Godot project is
+  EXPORT_NAME: Island
+
+jobs:
+  export-linux:
+    name: Linux Export
+    runs-on: ubuntu-latest
+    container:
+      image: barichello/godot-ci:mono-latest # not sure this image works with godot 3.4.4
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          lfs: true
+      - name: Resolve Dependencies
+        run: |
+          mkdir -v -p build/mods
+          cd $DIR
+          godot --export-pack "Linux/X11" --no-window ../build/mods/$EXPORT_NAME.pck
+      - name: Upload Artifact
+        uses: actions/upload-artifact@v1
+        with: # the final result should be called Island.zip
+          name: $EXPORT_NAME
```

Comment:
variable expansion works different in github actions (opposed to bash) so it would be `${{ EXPORT_NAME }}`